### DISCUSSION TITLE
GNOME cache

### DISCUSSION THREAD

**Discussion by PavanLuca** [bleachbit/bleachbit#1647] [Q&A] [answered]
Should I clean GNOME cache? Seems that it saves screenshots, icons and thumbnails that I view with GNOME Software. I can remove them if they're downloaded every time I open GNOME Software to recover space but I'd add GNOME Software cache path to exclusions if they're never refreshed to save connection.
![Schermata da 2024-03-13 18-23-43](https://github.com/bleachbit/bleachbit/assets/123581658/815bda48-3d0b-40d0-ab21-5972bb8350fb)

**Comment by az0** [comment, marked as answer]
Across many applications, deleting cache is generally safe. For network caches, the software re-downloads it as needed. It should work like this for GNOME Software too. 

For testing, you have a few options. 

1. You can delete all of it and see what happens
2. You can try to correlate specific screenshots with specific software you're browsing in GNOME Software. Use the filenames and file dates to correlate it. For example, I see the names of games in your screenshot. Close GNOME software, delete the screenshots for a specific game, and then browse that game in GNOME Software to check whether the screenshot comes back.
3. Close GNOME software, make a backup copy of `~/.config/gnome-software`, delete the original folder, and try the software again.

Does this help?

**Reply by PavanLuca** [reply]
Yes, done, clearing it doesn't break anything, the stuff is downloaded again when reopened, thanks 🤝.
